src/wattsi.pas

```diff
@@ -2600,29 +2626,52 @@ function Main(): Boolean;
          try
             try
                // gen...
-               for Variant in TVariants do
+               if (BuildType = 'default') then
                begin
-                  MkDir(OutputDirectory + '/multipage-' + kSuffixes[Variant]);
-                  Inform('Generating ' + Uppercase(kSuffixes[Variant]) + ' variant...');
-                  {$IFDEF TIMINGS} StartTime := Now(); {$ENDIF}
-                  ProcessDocument(Documents[Variant], Variant, BigTOC); // $R-
-                  {$IFDEF TIMINGS} Writeln('Elapsed time: ', MillisecondsBetween(StartTime, Now()), 'ms'); {$ENDIF}
-                  // output...
-                  if (Variant <> vDEV) then
+                  for Variant in TVariants do
```

Comment:
There's some copying of documents earlier on that also does that for all variants. I suppose we could add additional branching there too to save a bit of a processing time.